Inversion of the commit hash below to make musl slightly happier.

From 1c1bdb4f4aba988553a012c8cc0eb85b6e8aa662 Mon Sep 17 00:00:00 2001
From: Alex Shvartzkop <dudesuchamazing@gmail.com>
Date: Wed, 24 Apr 2024 00:43:01 +0300
Subject: [PATCH] Make mold slightly happier.

---
 utils/idftools/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/utils/idftools/CMakeLists.txt b/utils/idftools/CMakeLists.txt
index 64728d61eb4..8a0cb4a8bf2 100644
--- a/utils/idftools/CMakeLists.txt
+++ b/utils/idftools/CMakeLists.txt
@@ -31,7 +31,6 @@ target_link_libraries( dxf2idf
     dxflib_qcad
     idf3
     kimath
-    common
     ${OPENGL_LIBRARIES}
     ${wxWidgets_LIBRARIES}
     )
@@ -38,7 +39,6 @@ target_link_libraries( dxf2idf
 target_link_libraries( idf2vrml
     idf3
     kimath
-    common
     ${OPENGL_LIBRARIES}
     ${wxWidgets_LIBRARIES}
     )
-- 
GitLab

